docs/transports.md

```diff
@@ -467,6 +468,22 @@ PRs to this document are welcome for any new transports!
 + [pino-syslog](#pino-syslog)
 + [pino-websocket](#pino-websocket)
 
+<a id="@appsignal/nodejs-pino"></a>
+### @appsignal/nodejs
+
+[@appsignal/nodejs](https://www.npmjs.com/package/@appsignal/nodejs) [`AppsignalPinoTransport`](https://github.com/appsignal/appsignal-nodejs/blob/0fffae87ebe269bb86b6adafba23388ecb802979/src/pino_transport.ts) is the official [AppSignal](https://appsignal.com/) transport for Pino.
+
+```javascript
+import pino from 'pino';
+import { Appsignal, AppsignalPinoTransport } from '@appsignal/nodejs';
+
+const logger = pino({
+  AppsignalPinoTransport({
+    client: Appsignal.client,
```

Comment:
I don't think this would work with `pino-transport`

Reply:
Could you provide more info about how it wouldn't work with `pino-transport` please? I'm trying to find some info about it, but I haven't found anything.